CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.1
+* **feature** Migrated to Swift 4.2
+* **improvement** Added more devices to Device
```

Comment:
There should be some more according to [this list of commits](https://github.com/tbointeractive/bytes/commits/develop). Do you want to list those here as well?

Reply:
You're right. I've added those changes.